spec/models/jobseeker_profile_spec.rb

```diff
@@ -46,4 +46,36 @@
       end
     end
   end
+
+  describe "#replace_qualifications!" do
```

Comment:
My anxiety would make me create another qualification that was not associated with this profile so that we can assert that only the correct qualifications were deleted!